pkg/api/v1/schema.go

```diff
@@ -125,6 +125,7 @@ type CPU struct {
 	Cores uint32 `json:"cores,omitempty"`
 	// Model specifies the CPU model inside the VMI.
 	// List of available models https://github.com/libvirt/libvirt/blob/master/src/cpu/cpu_map.xml.
+	// You also can specify special cases like "passthrough" to get the same CPU as the node.
```

Comment:
Can you als add what happens by default?

Reply:
I think that we will also need support for `host-model` as extra `model`. To make the default explicit configurable too.

Reply:
done